### PR TITLE
Fix error status icons with tooltips: added screenreader and tabIndex 

### LINKED ISSUE #1155: [A11y] Bug - status icon not screenreader accessible
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Screens:

Miscellaneous 
status icon

As noted in [RHODS-6974](https://issues.redhat.com/browse/RHODS-6974) and [RHODS-6978](https://issues.redhat.com/browse/RHODS-6978)

### Expected Behavior

Icon should be VO accessible

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #1155 

## Description

Add screenreader and tabIndex to error status icons wrapped by tooltips. 

## How Has This Been Tested?

For status icons wrapped by tooltips, added icon to tab order and made icon screenreader accessible.  

## Request review criteria:

- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

### CODE REVIEW

File: frontend/src/pages/projects/components/StorageSizeBars.tsx [hunk 1]
```diff
@@ -43,8 +43,12 @@ const StorageSizeBar: React.FC<StorageSizeBarProps> = ({ pvc }) => {
   let inUseRender: React.ReactNode;
   if (error) {
     inUseRender = (
-      <Tooltip removeFindDomNode content={`Unable to get storage data. ${error.message}`}>
-        <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+      <Tooltip removeFindDomNode content={`Unable to get storage data.`}>
```
Comment (alexcreasy) [inline frontend/src/pages/projects/components/StorageSizeBars.tsx:46]: Was removing `error.message` from the tooltip intentional? Just checking :)

**Review by alexcreasy — COMMENTED** [review]
Code reviewed, looks good! The formatting issues the linter picked up need fixing before I can test it though :)

File: frontend/src/pages/projects/components/StorageSizeBars.tsx (hunk 1 above)
Comment (jenny-s51) [reply frontend/src/pages/projects/components/StorageSizeBars.tsx]: Nice catch, reverted

**Review by jenny-s51 — COMMENTED** [review]
Thank you @alexcreasy , updated

**Review by Gkrumbach07 — APPROVED** [review]
/lgtm

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm, I think we can merge this. Looks like the problem @alexcreasy pointed out was solved.

**Pull request merged**